lib/token.js

```diff
@@ -302,6 +302,18 @@ function handleOAuthResponse(sdk, oauthParams, res) {
     }
   }
 
+  if (res['code']) {
```

Comment:
Yeah, I'd forgotten that I used Array.isArray the last time I was poking through this code. I should just convert everything to Array.isArray, I have a polyfill anyway: https://github.com/okta/okta-auth-js/blob/master/lib/vendor/polyfills.js#L82.